File: January/Day4/minimum_rounds.py
class Solution:
    def minimumRounds(self, tasks: List[int]) -> int:
        c = Counter(tasks)
        count = 0
        for key, value in c.items():
            if value == 1:
                return -1
            if value%3 == 0:
                count += value//3
            else:
                count += value//3 + 1
        return count

File: January/Day4/minimumRounds.java
class Solution {
    public int minimumRounds(int[] tasks) {
        int count=0;
        HashMap<Integer,Integer> map = new HashMap<>();
        for(int n:tasks){
            map.put(n,map.getOrDefault(n,0)+1);
        }
        for (Map.Entry<Integer,Integer> mapElement : map.entrySet()) {
            Integer value=mapElement.getValue();
            if(value==1)return -1;
            else if(value==2)count++;
            else {
                count+=Math.ceil(value/3.0);
            }
            }
        return count;
    }
}
